flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointExecuteResult.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.checkpoint;
+
+/**
+ * The result of executing a checkpoint.
+ */
+public class CheckpointExecuteResult {
+
+	private final CompletedCheckpoint completedCheckpoint;
+
+	private final CheckpointAbortReason failure;
+
+	private final Throwable failureCause;
+
+	public CheckpointExecuteResult(CompletedCheckpoint success) {
```

Comment:
can be package private

Reply:
No, there are other places where use this class such as `CheckpointExecuteResult `.

Reply:
The current class is `CheckpointExecuteResult`, and all the usages are in `PendingCheckpoint`

Reply:
No, it is used in `JobManager.scala` and `JobMaster.java`

Reply:
Oh, I thought you were referring to the access modifier of this class. Yes, you are right.